jest.config.js

```diff
@@ -10,7 +10,6 @@ module.exports = {
       '<rootDir>/client/src/__mocks__/styleMock.js',
     // CSS Modules - match files that end with 'module.css'
     '\\.module\\.css$': 'identity-obj-proxy',
-    'react-i18next': '<rootDir>/client/src/__mocks__/react-i18nextMock.js',
```

Comment:
It took me a couple hours to figure this out as my test setup just kept failing.

I believe the purpose of this line is we want to mock `react-i18next` for testing. The issue with this is that:
1. Placing this under `moduleNameMapper` means asking Jest to stub the library globally, rather than just mocking it (which allows us to unmock)
2. For third-party libraries, the mock file should be named after the library that it is mocking, and Jest can automatically find them under `__mocks__`, so we wouldn't have to specify/map them a config.

What I did to address this was to rename `react-i18nextMock.js` to `react-i18next.js` to match the library name. Jest would now able to automatically search and mock the library, which in turn allows me to unmock it and use the actual implementation in my test.